eclipse-che/che-server
# Move dockerfiles to build/dockerfiles

### What does this PR do?
Moves dockerfiles from dockerfiles/che to build/dockerfiles
Removes the endpoint-watcher, postgres and keycloak images.
Adds brew.Dockerfile from downstream.

Coauthored with @SDawley 
### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

https://issues.redhat.com/browse/CRW-4043

### How to test this PR?

Run `./build/dockerfiles/build.sh` from the project's root directory.
See: docker image `quay.io/eclipse/che-server:next` has been built. 

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
